idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1712 @@
+            balance_type=MaterialBalanceType.componentTotal,
+            has_phase_equilibrium=False,
+            has_mass_transfer=True)
+
+        self.liquid_phase.add_energy_balances(
+            balance_type=EnergyBalanceType.enthalpyTotal,
+            has_heat_transfer=True)
+
+        self.liquid_phase.apply_transformation()
+
+        # Add Ports for vapor side
+        self.add_inlet_port(name="vapor_inlet", block=self.vapor_phase)
+        self.add_outlet_port(name="vapor_outlet", block=self.vapor_phase)
+
+        # Add Ports for liquid side
+        self.add_inlet_port(name="liquid_inlet", block=self.liquid_phase)
+        self.add_outlet_port(name="liquid_outlet", block=self.liquid_phase)
+
+    # ==========================================================================
+        """ Add performace equation method"""
+        self._make_performance()
+
+
+    def _make_performance(self):
+        """
+        Constraints for unit model.
+
+        Args: None
+
+        Returns: None
+
+        """
+
+        # ======================================================================
+        # Aliases for  Sets
+        vap_comp = self.config.vapor_side.property_package.component_list
+        liq_comp = self.config.liquid_side.property_package.component_list
+        dcomp = self.config.liquid_side.property_package.component_list_d
+        vapor_phase_list_ref = self.config.vapor_side.property_package.phase_list
+        liquid_phase_list_ref = self.config.liquid_side.property_package.phase_list
+
+        # Add object reference - time
+        add_object_reference(self,
+                             "t",
+                             self.flowsheet().config.time)
+
+        # Packing  parameters
+        self.eps_ref = Param(initialize=self.config.packing_void_fraction,
+                             units=None,
+                             doc="Packing void space m3/m3")
+
+        self.a_ref = Param(initialize=self.config.packing_specific_area,
+                           units=pyunits.m**2 / pyunits.m**3,
+                           doc="Packing specific surface area m2/m3")
+
+        self.dh_ref = Expression(expr= 4 * self.eps_ref /self.a_ref,
+                                 doc="Hydraulic diameter")
+
+        # specific constants for volumetric mass transfer coefficients
+        # reference:  Billet and Schultes, 1999
+        self.Cv_ref = Var(initialize=0.357,
+                          doc='''Vapor packing specific constant in
+                                Billet and Schultes' (1999) volumetric
+                                mass transfer coefficient correlation''')
+        self.Cl_ref = Var(initialize=0.5,
+                          doc='''Liquid packing specific constant in
+                                Billet and Schultes' (1999) volumetric
+                                mass transfer coefficient correlation''')
+        self.Cv_ref.fix()
+        self.Cl_ref.fix()
```

Comment:
@PTA0001  Why not this be a param if you are fixing the var? Fixing the var in the model is dangerous as users might not be aware this is being done.

Reply:
The question here is whether this is a parameter that a user might want to optimize for. If a user might want to optimize for this value, then it should be a `Var`, if it is something a user must always provide, then a `Param` is acceptable.

Reply:
Yes, it was made a ```var``` as this might be used for estimating the mass transfer coefficients.